docs/Extensibility_UG/frontend_extensions.md

```diff
@@ -191,11 +269,21 @@ The next example illustrates using ``ConversionExtension`` for conversion of “
          :fragment: [py_frontend_extension_ThresholdedReLU]
 
 
-To access original framework operation attribute value and connect to inputs, ``node`` object of type ``NodeContext`` is used. It has two main methods:
+The next example shows how to use ``ConversionExtension`` to convert Pytorch `aten::hardtanh <https://pytorch.org/docs/stable/generated/torch.nn.functional.hardtanh.html>`_ to demonstrate how to use ``get_values_from_const_input`` function to fetch an attribute value from input:
+
+
+.. doxygensnippet:: docs/snippets/ov_extensions.py
+   :language: python
+   :fragment: [py_frontend_extension_aten_hardtanh]
+
+
+To access original framework operation attribute value and connect to inputs, ``node`` object of type ``NodeContext`` is used. It has three main methods:
 
 * ``NodeContext::get_input`` to get input with a given index,
 
-* ``NodeContext::get_attribute`` to get attribute value with a given name.
+* ``NodeContext::get_attribute`` to get attribute value with a given name,
+
+* ``NodeContext::get_values_from_const_input`` to get an attribute with a given input index.
 
```

Comment:
@ilya-lavrenov, @slyalin, should we have a link to openvino_contrib reporsitory in this documentation where we have samples of extensions?

Reply:
good suggestion

Reply:
@mateusztabaka, please cover this in this PR.

Reply:
done